devtools/platform-descriptor-json/src/main/resources/codestarts/optaplanner-example/java/src/main/java/org/acme/optaplanner/bootstrap/DemoDataGenerator.java

```diff
@@ -0,0 +1,194 @@
+package org.acme.optaplanner.bootstrap;
```

Comment:
shouldn't this file and whole optaplanner example be pushed into optaplanner extension rather than in the bunlded/platform level  descriptors ?

Reply:
Of course, same for Qute and RestEasy but "During phase 1 (and until the api is more stable), all the codestarts are embeded in quarkus (and not yet part of extensions)". I chose to implement this extension because I knew it was a bit complex therefore good to catch problems..